# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: dom/canvas/crashtests/crashtests.list
# Commit: c48f0d2c6c1d
# Full Hash: c48f0d2c6c1d3a61945193cffc22ad20533b12b7
# Author: Lee Salzman <lsalzman@mozilla.com>
# Date: 2025-07-09 03:16:15
# Description:
#   Bug 1975981 - Move optimized filter check later. r=aosmond
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D256330
# ==============================================================================

diff -r 6295a834a93d -r c48f0d2c6c1d dom/canvas/crashtests/crashtests.list
--- a/dom/canvas/crashtests/crashtests.list	Tue Jul 08 17:21:50 2025 +0000
+++ b/dom/canvas/crashtests/crashtests.list	Tue Jul 08 17:30:19 2025 +0000
@@ -69,3 +69,4 @@
 load 1857986.html
 load 1873790.html
 load 1961191.html
+load 1975981-1.html
